core/domain/exp_jobs_one_off.py

```diff
@@ -535,3 +536,41 @@ def reduce(key, values):
         # Combine all values from multiple lists into a single list
         # for that error type.
         yield (key, list(set().union(*final_values)))
+
+
+class ImageDataMigrationJob(jobs.BaseMapReduceOneOffJobManager):
+    """ One-off job for migrating the images in the exploration
```

Comment:
Follow the usual docstring conventions. No space after the opening """; deindent the next line by 4; end with a period.

/cc @apb7 @kevinlee12 -- this should be caught by the linter.

Reply:
Done!